specification/paths/HookLogs-hook_log_id.json

```diff
@@ -0,0 +1,52 @@
+{
+  "get": {
+    "tags": [
+      "HookLogs"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "experimental"
+        ]
+      }
+    ],
+    "summary": "Get a hook log by id.",
+    "description": "This endpoint retrieves the hook log with the supplied id.",
+    "parameters": [
+      {
+        "$ref": "#/components/parameters/path-hook-log_id"
+      }
+    ],
+    "responses": {
+      "200": {
+        "description": "Retrieved the hook log.",
+        "content": {
+          "application/vnd.api+json": {
+            "schema": {
+              "type": "object",
+              "required": [
+                "data"
+              ],
+              "additionalProperties": false,
+              "properties": {
+                "data": {
+                  "allOf": [
+                    {
+                      "$ref": "#/components/schemas/HookLog"
+                    },
+                    {
+                      "required": [
```

Comment:
You can add "id" and "links" into the schema since they will always be there. We will never add a post hooklogs endpoint so there will never be a case that we will not have an id or links for this resource.